podcast/2.13.md

```diff
@@ -0,0 +1,913 @@
+---
+title: 'Building technical communities, with swyx, Head of Developer Experience at Temporal'
+publishDate: 2021-12-14T10:00-07:00
+tags: [podcast]
+slug: swyx
+published: true
+---
+
+<!-- START AUDIO -->
+
+<audio className="object-center" src="https://www.buzzsprout.com/1097978/9593278-arming-the-rebels-of-the-metaverse-with-joseph-nelson-ceo-and-co-founder-of-roboflow.mp3?download=true" controls preload="none"></audio>
```

Comment:
To be updated.

Reply:
@nickmyyz https://www.buzzsprout.com/1097978/9653528-building-technical-communities-with-swyx-head-of-developer-experience-at-temporal.mp3?download=true